src/inspirehep-holdingpen-js/holdingpen/holdingpen.services.js

```diff
@@ -146,6 +153,13 @@
     
               var _extra_data = workflow._extra_data;
               return _extra_data && _extra_data.validation_errors && _extra_data.validation_errors.length > 0;
+            },
+
+            hasCrawlErrors: function (workflow) {
+              if (!workflow) { return false; }
+    
+              var _extra_data = workflow._extra_data;
```

Comment:
here it we might already have an ``attribute`` error

Reply:
what do you mean?